packages/loot-design/src/components/hooks.js

```diff
@@ -13,3 +15,10 @@ export function useScrollFlasher() {
 
   return scrollRef;
 }
+
+export function useSetThemeColor(color, opts) {
+  useEffect(() => {
+    if (opts && opts.skip) return;
+    setThemeColor(color);
+  });
```

Comment:
🤔 Hmm, I'm trying but not seeing this bug. 
The `useEffect` wrapper is due to us touching stuff outside the React render tree, and with no deps array it will fire on every call to `useSetThemeColor`. What am I missing?

Reply:
Generally this would have bene caught by `react-hooks/exhaustive-deps` eslint rule (which will be added once we upgrade eslint). Either way, not a big thing and definitely not blocking the release :)

Some more info: https://bobbyhadz.com/blog/react-hooks-exhaustive-deps